src/3d/symbols/qgspoint3dsymbol.h

```diff
@@ -96,6 +106,7 @@ class _3D_EXPORT QgsPoint3DSymbol : public QgsAbstract3DSymbol
     Shape mShape = Cylinder;  //!< What kind of shape to use
     QVariantMap mShapeProperties;  //!< Key-value dictionary of shape's properties (different keys for each shape)
     QMatrix4x4 mTransform;  //!< Transform of individual instanced models
+    QgsMarkerSymbol *mBillboardSymbol = nullptr;
```

Comment:
Does this have ownership? If so, it should be a unique_ptr

Reply:
Changed to use `unique_ptr`. I need to re-implement the `clone` function to do this. I am not quite sure if my approach is good or not.